src/main/java/nl/tudelft/ewi/devhub/server/database/AssignTAs.java

```diff
@@ -0,0 +1,54 @@
+package nl.tudelft.ewi.devhub.server.database;
+
+import nl.tudelft.ewi.devhub.server.database.entities.AssignedTA;
+import nl.tudelft.ewi.devhub.server.database.entities.Assignment;
+import nl.tudelft.ewi.devhub.server.database.entities.Group;
+import nl.tudelft.ewi.devhub.server.database.entities.User;
+
+import java.util.*;
+
+import static com.google.common.collect.Lists.newArrayList;
+import static com.google.common.collect.Lists.partition;
+import static java.util.Collections.shuffle;
+
+/**
+ * Created by sayra on 07/06/2017.
+ */
+public class AssignTAs {
+    public static List<AssignedTA> assignGroups(Collection<User> TAs, Collection<Group> groupList, Assignment assignment, Random random) {
+        List<AssignedTA> assignedTAs = new ArrayList<>();
+
+        List<Group> groups = new ArrayList<>();
+        groups.addAll(groupList);
```

Comment:
You can do this directly with the [`Collection` constructor](https://docs.oracle.com/javase/7/docs/api/java/util/ArrayList.html#ArrayList(java.util.Collection))

```java
List<Group> groups = new ArrayList<>(groupList);
```